Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -120,6 +120,11 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
 
     float documentFittingScale() const { return m_documentLayout.scale(); }
 
+#if PLATFORM(MAC)
+    WebCore::FloatRect convertFromPDFPageToScreenForAccessibility(const WebCore::FloatRect&, PDFDocumentLayout::PageIndex) const;
+    void accessibilityScrollToPage(int);
```

Comment:
Can you address Simon's review comment here? This should be a `PDFDocumentLayout::PageIndex` instead of an `int`.